webvtt.go

```diff
@@ -440,7 +445,19 @@ func (s Subtitles) WriteToWebVTT(o io.Writer) (err error) {
 
 	// Add header
 	var c []byte
-	c = append(c, []byte("WEBVTT\n\n")...)
+	c = append(c, []byte("WEBVTT")...)
+
+	// Write X-TIMESTAMP-MAP if set
+	if s.Metadata != nil {
+		if s.Metadata.WebVTTXTimestampMap != nil {
+			c = append(c, []byte("\n")...)
+			mpegts := fmt.Sprintf("MPEGTS:%d", s.Metadata.WebVTTXTimestampMap.PresentationTimestamp)
```

Comment:
Could you create a `func (m *WebVTTTimestampMap) String() string` that would return the header string instead, and use this function here?

That would only leave `c = append(c, append([]byte("\n"), []byte(s.Metadata.WebVTTXTimestampMap.String())...)...)` here